docker/cajtck.sh

```diff
@@ -48,24 +54,23 @@ cd $TS_HOME/bin
 sed -i "s#^build.level=.*#build.level=2#g" ts.jte
 sed -i "s#^endorsed.dirs=.*#endorsed.dirs=$TCK_HOME/glassfish5/glassfish/modules/endorsed#g" ts.jte
 sed -i "s#^local.classes=.*#local.classes=$TCK_HOME/glassfish5/glassfish/modules/endorsed/jakarta.annotation-api.jar#g" ts.jte
-sed -i "s#^report.dir=.*#report.dir=$TCK_HOME/cajtckreport/cajtck#g" ts.jte
-sed -i "s#^work.dir=.*#work.dir=$TCK_HOME/cajtckwork/cajtck#g" ts.jte
+sed -i "s#^report.dir=.*#report.dir=$TCK_HOME/${TCK_NAME}report/${TCK_NAME}#g" ts.jte
+sed -i "s#^work.dir=.*#work.dir=$TCK_HOME/${TCK_NAME}work/${TCK_NAME}#g" ts.jte
```

Comment:
A nit, but I would probably add a dash in front of "report" and "work".

Reply:
Actually, why does TCK_NAME need to appear before "report" or "work" at all
since it occurs in the directory name after "report" and "work"?  For that matter,
why does it need to appear in the directory name at all?  Maybe it's needed to
allow multiple TCKs to run in the same directory?

For Jakarta Mail, the directories are JTreport and JTwork, with no TCK_NAME
in the directory name at all.

Reply:
> Actually, why does TCK_NAME need to appear before "report" or "work" at all
> since it occurs in the directory name after "report" and "work"? For that matter,
> why does it need to appear in the directory name at all? Maybe it's needed to
> allow multiple TCKs to run in the same directory?
${TCK_NAME}report is required when the TCKs share the same workspace. ${TCK_NAME}report/${TCK_NAME} may not be required, but let me keep it for now.
> 
> For Jakarta Mail, the directories are JTreport and JTwork, with no TCK_NAME
> in the directory name at all.